canonical-web-and-design/ubuntu.com
# Build /kubernetes/compare page

## Done

- Build new `/kubernetes/compare` page 
- Add to secondary nav
- Add meta image

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/kubernetes/compare
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copy doc](https://docs.google.com/document/d/1JngYmRhohPo62yU72V1xpzCCILI21SmHKKttBxHjPog/edit?ts=5faa80b7)

**NOTE**: A new pattern was made for the ratings meter, as the l-tutorials-meter which is used on the tutorials page 1) has a very specific class name and 2) doesn't allow for ratings which aren't whole numbers.  Should we combine these two patterns somehow using a more generic class name? 


## Issue / Card

Fixes #8680 

## Screenshots

![image](https://user-images.githubusercontent.com/58959073/99090671-d4455d00-25c6-11eb-80b5-8cba95572453.png)
